index.js

```diff
@@ -738,3 +746,7 @@ function zigZagEncodeBigInt (n) {
   // 0, -1, 1, -2, 2, ...
   return n < 0n ? (2n * -n) - 1n : n === 0n ? 0n : 2n * n
 }
+
+function validateUint (n) {
+  if ((n >= 0) === false) throw new Error('uint must be positive')
```

Comment:
```suggestion
  if ((n >= 0) === false /* Handles NaN as well */) throw new Error('uint must be positive')
```

To make it perfectly clear for our future selves.